gr-qtgui/grc/qtgui_entry.block.yml

```diff
@@ -21,6 +21,13 @@ parameters:
     label: Default Value
     dtype: ${ type }
     default: '0'
+-   id: entry_signal
+    label: Update trigger
+    dtype: enum
+    default: editingFinished
+    options: [returnPressed, editingFinished]
+    option_labels: ["'Enter' is pressed", "'Enter' is pressed / line edit loses focus"]
```

Comment:
Maybe "Enter", "Enter or focus lost" to make it a little shorter.

Nit: "Update Trigger" (caps) to match other param names.